qiskit/opflow/operator_base.py

```diff
@@ -47,6 +47,11 @@ class OperatorBase(StarAlgebraMixin, TensorMixin, ABC):
     def __init__(self) -> None:
         self._instance_id = next(self._count)
 
+    @property
+    def settings(self) -> Dict:
+        """Return settings."""
+        return {}
```

Comment:
I imagine this is intended to be overridden - if a sub-class does not and this  ends up getting called its not going to be meaningful. Should this property be made abstract - which would force any sub-class to implement it to become concrete and thus instantiable, or alternative if not instead change the default behavior to raise an error rather than returning and empty dict.

Reply:
Fixed in 0dfcc3b